docs/source/generics.rst

```diff
@@ -806,20 +804,18 @@ variable is invariant:
 
 .. code-block:: python
 
-   from typing import TypeVar
-   from typing_extensions import Protocol
+   from typing import Protocol, TypeVar
```

Comment:
I would leave this be since we still support 3.7 which doesn't have typing.Protocol.